bin/_mocha

```diff
@@ -199,7 +205,8 @@ program
   .option('--delay', 'wait for async suite definition')
   .option('--allow-uncaught', 'enable uncaught errors to propagate')
   .option('--forbid-only', 'causes test marked with only to fail the suite')
-  .option('--forbid-pending', 'causes pending tests and test marked with skip to fail the suite');
+  .option('--forbid-pending', 'causes pending tests and test marked with skip to fail the suite')
+  .option('--file <file>', 'include a file to be ran during the suite', list, []);
```

Comment:
alas, we `-f` is taken, so no alias is possible

Reply:
I would prefer this was a repeatable flag instead of one that must be comma-delimited.  I assume that's possible in [commander](https://npm.im/commander), but am unfamiliar with its API